system/mrm_handler/include/mrm_handler/mrm_handler_core.hpp

```diff
@@ -38,6 +38,9 @@
 #include <diagnostic_msgs/msg/diagnostic_array.hpp>
 #include <nav_msgs/msg/odometry.hpp>
 
+#define CALL true
+#define CANCEL false
```

Comment:
It looks you had better use enum like `RequestType`.

Reply:
It is modified to use enum.
Request types like ` CALL` and `CANCEL` should be written so that they are explicitly known on the caller side.
Just for readability...